### DISCUSSION TITLE
setVolume(0.0)

### DISCUSSION THREAD

**Discussion by spairo1741** [pschatzmann/arduino-espeak-ng#22] [Q&A]
hi, 
It works on the Arduino system version 2.3.2, the board is liLyGo T-embed with esp32S3, esp32 boards is version 3.05. 
I run the espeak-arduino program with the Polish language on this set, it works properly. 
I added #include "AudioTools/VolumeStream.h", 
VolumeStream volume(i2s); 
and in setup 
// setup output
  auto espeak_info = espeak.audioInfo();
  auto cfg = i2s.defaultConfig();
  cfg.channels = espeak_info.channels; 							// 1
  cfg.sample_rate = espeak_info.sample_rate; 				// 22050
  cfg.bits_per_sample = espeak_info.bits_per_sample;                 // 16
  i2s.begin(cfg);
		// set initial volume
  volume.begin(cfg); 	// we need to provide the bits_per_sample and channels
  volume.setVolume(0.0);								// volume!	
  espeak.say("Good morning!");
 for(;;);
But I still hear "good morning" after restart

**Comment by pschatzmann** [comment]
I guess you forgot to change the output of espeak from i2s to volume!
